### ISSUE TITLE
Proposal: Add function NewFromFloat

### ISSUE THREAD

**Issue by npinochet** [Rhymond/go-money#108]
It would be really useful to be able to create a new Money struct from a float or maybe a decimal string ('1.243'). Formatting the decimal amount from a source (API response, file, ...) to a Money struct can get really tedious, that's why I propose a util function inside the money package for dealing with this cases.

This is my current implementation:
```go
func NewFromFloat(amount float64, currency string) *money.Money {
	currencyDecimals := math.Pow(10, float64(money.GetCurrency(currency).Fraction))
	amountCents := int64(amount * currencyDecimals)
	return money.New(amountCents, currency)
}
```
To not lose precision dealing with floats there can also be a NewFromString, but I haven gotten around to implementing it.

What do you guys think?

**Comment by Rhymond** [comment]
Hi @npinochet, great idea! I would love to see it implemented, I cannot find any issues with your code example

**Comment by npinochet** [comment]
Perfect, I went ahead and made the PR https://github.com/Rhymond/go-money/pull/115

**Comment by azlancpool** [comment]
Hi @Rhymond, first of all thank you so much for this great package!
When will we have a new release with these changes?

**Comment by Rhymond** [comment]
Hey @azlancpool, Thanks! I just reviewed a PR and left couple of comments, if you're interested in `NewFromFloat` that's probably going to be merged soon.

**Comment by Rhymond** [comment]
@azlancpool NewFromFloat merged and released in v1.0.9

**Comment by npinochet** [comment]
As this issue is only for NewFromFloat, I'll close the issue since it's done 👍
